mods/fire/init.lua

```diff
@@ -3,7 +3,12 @@
 -- Global namespace for functions
 
 fire = {}
-
+fire.features = {
```

Comment:
It provides a means for other mods to determine if they can use the newer Fire feature on_burn callback, by checking this (as opposed to using a version number).

This allows mods to save adding a custom ABM for burning nodes that the mod wishes to turn into a charred version(for example).  It also provides easy and meaningful feature checking of the Fire API present, as opposed to using a version number check.  It was suggested by celeron55 as a better alternative.